docs/backends/bumble.rst

```diff
@@ -0,0 +1,116 @@
+Bumble backend
+===============
+
+This backend adds support for the |bumble| Bluetooth Controller Stack from Google.
+The backend enables support of multiple |bumble_transport| to communicate with
+a physical or virtual HCI controller.
+
+Use cases for this backend are:
```

Comment:
I think the serial/USB use case is worth mentioning here too. (Probably as number 1)